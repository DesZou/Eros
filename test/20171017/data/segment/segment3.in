10 10 10
73473 25961 18943 69301 77558 53040 34887 51110 21033 95323 
2 5 7 73142
2 4 4 2672
1 2 6 93
2 5 10 78228
1 3 6 61
2 5 9 49971
2 8 9 68511
1 3 6 95
1 3 6 4
1 4 9 29
8 9
2 3
2 3
5 7
5 9
7 8
1 10
4 10
5 10
3 8
