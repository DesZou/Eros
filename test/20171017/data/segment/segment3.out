131154.500
45250.000
45250.000
201489.250
332643.750
132698.167
532407.750
413684.750
410871.750
289075.500
